Quotient/csapi/whoami.cpp

```diff
@@ -17,4 +17,5 @@ GetTokenOwnerJob::GetTokenOwnerJob()
               makePath("/_matrix/client/v3", "/account/whoami"))
 {
     addExpectedKey("user_id");
+    setMaxRetries(std::numeric_limits<int>::max());
```

Comment:
This is generated code (look in `CONTRIBUTING.md` and search `csapi` in it) - this line will be overwritten the next time I run the generator (which can be almost any time).